pilot/proxy/envoy/testdata/fault-route-v1alpha2.yaml.golden

```diff
@@ -0,0 +1,26 @@
+hosts:
+  - world
+http:
+  - match:
+      - headers:
+          scooby:
+            exact: doo
+          animal:
+            prefix: dog.cat
+          name:
+            regex: "sco+do+"
+        sourceLabels:
+            version: v0
```

Comment:
can you add a real multi match? I just want to see multiple routes being generated.

Reply:
added it to `multi-match-fault-v1alpha2.yaml.golden`